packages/editor/src/client/blocks/teaserGridBlock.tsx

```diff
@@ -332,29 +351,16 @@ export function TeaserContent({
         <Box display="flex" flexWrap="wrap">
           <Box flexShrink={0} marginRight={Spacing.ExtraSmall}>
             <Typography variant="subtitle1" color="gray">
-              Style: {labelForTeaserStyle(style)}
+              {labelForTeaserStyle(style)}
             </Typography>
           </Box>
           <Box flexShrink={0}>
             <Typography variant="subtitle1" color="gray">
-              Status: {states?.join(' / ')}
+              {states?.join(' / ')}
```

Comment:
What happened to `Status:`? Here you can work with two translations.

Reply:
I've included it directly in the translation `"state_published": "Status: Published"` - but when taking a closer look at the code now I wonder if this means that a post can have two states at the same time such as "published" and "pending" ? Then of course what I made doesn't work